templates/serverpod_templates/github/workflows/deployment-docker.yml

```diff
@@ -0,0 +1,180 @@
+          # the same host, you need to adjust the traefik configuration in the
+          # docker-compose.production.yaml file.
+          SERVERPOD_API_SERVER_PUBLIC_PORT: 8080
+          # The public scheme (http/https) for the API server
+          SERVERPOD_API_SERVER_PUBLIC_SCHEME: "https"
+          # The public host address of the Insights server
+          SERVERPOD_INSIGHTS_SERVER_PUBLIC_HOST: ${{ secrets.SERVERPOD_INSIGHTS_SERVER_PUBLIC_HOST }}
+          # The public port number for the Insights server
+          # This is hardcoded, since it is used only to map from docker image to
+          # traefik loadbalancer and this port is NOT used to access from the
+          # web. External access is done solely through the host. If you want to
+          # have access through a different port or have multiple services on
+          # the same host, you need to adjust the traefik configuration in the
+          # docker-compose.production.yaml file.
+          SERVERPOD_INSIGHTS_SERVER_PUBLIC_PORT: 8081
+          # The public scheme (http/https) for the Insights server
+          SERVERPOD_INSIGHTS_SERVER_PUBLIC_SCHEME: "https"
+          # The public host address of the Web server
+          SERVERPOD_WEB_SERVER_PUBLIC_HOST: ${{ secrets.SERVERPOD_WEB_SERVER_PUBLIC_HOST }}
+          # The public port number for the Web server
+          # This is hardcoded, since it is used only to map from docker image to
+          # traefik loadbalancer and this port is NOT used to access from the
+          # web. External access is done solely through the host. If you want to
+          # have access through a different port or have multiple services on
+          # the same host, you need to adjust the traefik configuration in the
+          # docker-compose.production.yaml file.
+          SERVERPOD_WEB_SERVER_PUBLIC_PORT: 8082
+          # The public scheme (http/https) for the Web server
+          SERVERPOD_WEB_SERVER_PUBLIC_SCHEME: "https"
+          # The host address of the database
+          # This is hardcoded since the connection is done internally through
+          # the docker network. If you want to have access from the outside, you
+          # need to connect to your server with ssh
+          SERVERPOD_DATABASE_HOST: postgres
+          # The port number for the database connection
+          # If you need to change this port, you will also have to adjust
+          # postgres configuration in the docker-compose.production.yaml file
+          SERVERPOD_DATABASE_PORT: 5432
+          # Indicates if SSL is required for the database
+          # Set to false since it is behind the firewall and the database can
+          # only be accessed by utilizing the SSH tunnel
+          SERVERPOD_DATABASE_REQUIRE_SSL: false
+          # Specifies if the database connection is a Unix socket
+          SERVERPOD_DATABASE_IS_UNIX_SOCKET: false
+          # Redis is not required for this setup
+          SERVERPOD_REDIS_ENABLED: false
```

Comment:
Suggestion: I believe if this is redundant and if supplied you have to supply the full config for redis. Leaving it out will disable redis.

(We need to improve the loading of this config)